includes/Classifai/Services/Service.php

```diff
@@ -121,14 +123,22 @@ public function render_settings_page() {
 			?>
 			<div class="classifai-wrap wrap wrap--nlu">
 				<h2><?php echo esc_html( $this->display_name ); ?></h2>
-				<?php if ( ! empty( $this->provider_classes ) ) : ?>
+
+				<?php
+				if ( empty( $this->provider_classes ) ) {
+					echo '<p>' . esc_html__( 'No providers available for this service.', 'classifai' ) . '</p>';
```

Comment:
```suggestion
					echo '<p>' . esc_html__( 'No providers available for this service.', 'classifai' ) . '</p>';
					echo '</div></div>';
```
An optional nitpick: Adding closing `div` tags to prevent any potential UI conflicts.